Clinical-Genomics/cgweb
# Made mic,met and balsamic customer orderable

All ordertypes should now be available to pick for customers (not only by admin users)

How to test:
0. install on cgweb on clinical-db stage
1. go to the OP
1. login as an ordinary user (not as a user with admin access)
1. choose new order
1. open the ordertype drop down

Expected outcome:
In the ordertype drop down these option should also be available:

- metagenome
- mip
- balsamic
- mip + balsamic
